src/pages/admin/AdminPage.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+
+import { Layout, Menu, Input, Avatar, Badge } from 'antd';
+import {
+  UserOutlined,
+  PieChartOutlined,
+  DesktopOutlined,
+  FileOutlined,
+  TeamOutlined,
+  BellOutlined,
+  MailOutlined,
+} from '@ant-design/icons';
+
+import LogoAdmin from '../../assets/images/adidas-logo.png';
+
+import 'antd/dist/antd.css';
+import './AdminPage.scss';
+import { Link } from 'react-router-dom';
+
+const { Header, Content, Footer, Sider } = Layout;
+const { SubMenu } = Menu;
+const { Search } = Input;
+
+const AdminPage = () => {
+  const [collapsed, setCollapsed] = useState(false);
+
+  const toggleCollapsed = () => {
+    setCollapsed(!collapsed);
+  };
+
+  return (
+    <Layout className='layout-page'>
+      <Sider
+        collapsible
+        collapsed={collapsed}
+        onCollapse={toggleCollapsed}
+        className='sidebar'
+      >
+        <div className='logo'>
+          <Link to='/'>
+            <img alt='logo-admin' src={LogoAdmin} className='logo-admin' />
+          </Link>
+        </div>
+
+        <div className='line-bar'></div>
+        <Menu
+          defaultSelectedKeys={['1']}
+          mode='inline'
+          className='sidebar-menu'
+        >
+          <Menu.Item key='1' icon={<PieChartOutlined />} className='admin'>
+            <span className='title-admin'>Dashboard</span>
+          </Menu.Item>
+          <Menu.Item key='2' icon={<DesktopOutlined />}>
+            Option 2
+          </Menu.Item>
+          <SubMenu key='sub1' icon={<UserOutlined />} title='User'>
+            <Menu.Item key='3'>Tom</Menu.Item>
+            <Menu.Item key='4'>Bill</Menu.Item>
+            <Menu.Item key='5'>Alex</Menu.Item>
+          </SubMenu>
+          <SubMenu key='sub2' icon={<TeamOutlined />} title='Team'>
+            <Menu.Item key='6'>Team 1</Menu.Item>
+            <Menu.Item key='8'>Team 2</Menu.Item>
+          </SubMenu>
+          <Menu.Item key='9' icon={<FileOutlined />} />
+        </Menu>
+      </Sider>
+      <Layout className='site-layout'>
+        <Header className='site-layout-background'>
```

Comment:
Why component `Header` has className `site-layout-background` ??? It should be `header-layout` or something more relative

Reply:
@papercoding Fixed and changed to "header-layout"